daprdocs/content/en/developing-applications/building-blocks/state-management/howto-stateful-service.md

```diff
@@ -76,24 +74,31 @@ dapr_state_url = "http://localhost:3500/v1.0/state/{}".format(store_name)
 response = requests.delete(dapr_state_url + "/key1", headers={"consistency":"strong"})
 ```
 
-Last-write concurrency is the default concurrency mode if the `concurrency` option is not specified.
+If the `concurrency` option hasn't been specified, the default is last-write concurrency mode.
+
+## First-write-wins and last-write-wins
+
+Dapr allows developers to opt-in for two common concurrency patterns when working with data stores: 
 
-## First-write-wins and Last-write-wins
+- **First-write-wins**: useful in situations where you have multiple instances of an application, all writing to the same key concurrently.
+- **Last-write-wins**: Default mode for Dapr.
 
-Dapr allows developers to opt-in for two common concurrency patterns when working with data stores: First-write-wins and Last-write-wins.
-First-Write-Wins is useful in situations where you have multiple instances of an application, all writing to the same key concurrently.
+Dapr uses version numbers to determine whether a specific key has been updated. You can:
 
-The default mode for Dapr is Last-write-wins.
+1. Retain the version number when reading the data for a key.
+1. Use the version number during updates such as writes and deletes. 
 
-Dapr uses version numbers to determine whether a specific key has been updated. Clients retain the version number when reading the data for a key and then use the version number during updates such as writes and deletes. If the version information has changed since the client retrieved, an error is thrown, which then requires the client to perform a read again to get the latest version information and state.
+If the version information has changed since the version number was retrieved, an error is thrown, requiring the you to perform another read to get the latest version information and state.
 
-Dapr utilizes ETags to determine the state's version number. ETags are returned from state requests in an `ETag` header.
+Dapr utilizes ETags to determine the state's version number. ETags are returned from state requests in an `ETag` header. Using ETags, you know that a resource has been updated since the last time they checked by erroring during an ETag mismatch.
```

Comment:
```suggestion
Dapr utilizes ETags to determine the state's version number. ETags are returned from state requests in an `ETag` header. Using ETags, your application knows that a resource has been updated since the last time they checked by erroring during an ETag mismatch.
```